dspace-api/src/main/java/org/dspace/scripts/filepreview/FilePreview.java

```diff
@@ -101,30 +102,8 @@ public void internalRun() throws Exception {
 
         Context context = new Context();
         try {
-            if (StringUtils.isBlank(email)) {
-                handler.logError("Email is required for authentication.");
-                return;
-            }
-            if (StringUtils.isBlank(password)) {
-                handler.logError("Password is required for authentication.");
-                return;
-            }
-
-            EPerson eperson = ePersonService.findByEmail(context, email);
-            if (eperson == null) {
-                handler.logError("No EPerson found for this email: " + email);
-                return;
-            }
-
-            int authenticated = authenticateService.authenticate(context, email, password, null, null);
-            if (AuthenticationMethod.SUCCESS != authenticated) {
-                handler.logError("Authentication failed for email: " + email);
-                return;
-            } else {
-                handler.logInfo("Authentication successful for email: " + email);
-            }
-
-            context.setCurrentUser(eperson);
+            context.setCurrentUser(getEperson((context)));
```

Comment:
getEperson could have parameters email and password

Reply:
I don't understand. The method getEperson sets the current user to the one running the process, or it checks if the ePerson can be authorized based on the entered email and password (from cmd) and assigns it to this person.